src/test/resources/config/application-artemis.yml

```diff
@@ -21,6 +21,9 @@ artemis:
         internal-admin:
             username: artemis_admin
             password: artemis_admin
+        registration:
+            enabled: true
+            allowed-email-pattern: ([a-zA-Z0-9_\-\.\+]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})
```

Comment:
```suggestion
            allowed-email-pattern: [a-zA-Z0-9_\-\.\+]+@[a-zA-Z0-9_\-\.]+\.[a-zA-Z]{2,5}
```

Reply:
if we write it like this, we have to put it into `'...'`